tendrl-notifier.spec

```diff
@@ -10,6 +10,7 @@ URL: https://github.com/Tendrl/Notifier
 BuildRequires: pytest
 BuildRequires: systemd
 BuildRequires: python-mock
+BuildRequires: pysnmp
```

Comment:
pysnmp package available on centos/epel/rhel? 

cc: @TimothyAsir

Reply:
@r0h4n @shtripat @anmolbabu  if i install pysnmp uisng yum it does not have all classes which i want for snmp notification, if i install pysnmp using easy_install then it gives all features. pysnmp package is differ from these two way of installation. I have used "pysnmp.hlapi"  in import, But it gives import error when I installed using yum install pysnmp but it is available when i install using easy_install pysnmp.

but easy_install pysnmp needs gcc and python-devel as dependency.

Reply:
yum, installation is installing pysnmp package with version pysnmp-4.2.5-2.el7.noarch but it is not latest one. epel  is not giving the latest version of pysnmp. Old version does not give sufficient SNMP classes to send SNMP notification. It is available only in a new version. So we need pysnmp new version of pysnmp 4.3.9.  But it is not available in EPEL. 

but it is available in website link : http://dl.fedoraproject.org/pub/fedora/linux/updates/26/x86_64/p/python2-pysnmp-4.3.9-1.fc26.noarch.rpm
@TimothyAsir @r0h4n @TimothyAsir @shtripat @nthomas-redhat please give your suggestions

Reply:
@GowthamShanmugam , If that is the case, we will build this latest version and keep it in our dependencies repo. So that it will come automatically from our repo. We will remove it once its comes to EPEL.

Reply:
@GowthamShanmugam Please let me know the required dependency versions of the packages
pysmi
pycryptodome
pyasn1

Reply:
@TimothyAsir I remember @TimothyAsir suggested to get the built and serve from our repo.